gammapy/cube/core.py

```diff
@@ -324,6 +320,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap along slicepos`.
```

Comment:
For `SkyMap`, there is a `.copy()` method, maybe it would make sense to have that for `SkyCube` as well? Then one could copy the cube before creating the `SkyMap`. But there should definitely be the option `copy=False`, that gives you a reference ('view') to the data. Which is useful, if you want to access large data with the `SkyImageList` interface instead of `SkyCube`.